tests/unit/core/test_searchUtils.js

```diff
@@ -1,100 +1,64 @@
-import createStore from 'amo/store';
-import * as searchActions from 'core/actions/search';
-import * as api from 'core/api';
-import { initialApiState } from 'core/reducers/api';
-import { ADDON_TYPE_THEME } from 'core/constants';
-import { loadByCategoryIfNeeded, mapStateToProps } from 'core/searchUtils';
+import {
+  isLoaded,
+  parsePage,
+} from 'core/searchUtils';
 
 
-describe('searchUtils mapStateToProps()', () => {
+describe('Search.isLoaded()', () => {
   const state = {
-    api: { lang: 'fr-CA' },
-    search: {
-      filters: { clientApp: 'firefox', query: 'foo' },
-      hasSearchParams: true,
-    },
+    page: 2,
+    filters: { query: 'ad-block' },
+    loading: false,
+    results: [{ slug: 'ab', name: 'ad-block' }],
```

Comment:
Since you already have:

https://github.com/mozilla/addons-frontend/blob/e43b9acdca9d903aab38310aa1764a3b8411c68e/tests/unit/amo/helpers.js#L103

It might not be hard to convert this test suite to dispatched/reduced state rather than hard-coded state. If it starts taking a lot of time then don't worry about it.